esm/interpreter/pyodide.js

```diff
@@ -2,6 +2,7 @@ import { fetchPaths, stdio, writeFile } from './_utils.js';
 import { registerJSModule, run, runAsync, runEvent } from './_python.js';
 
 const type = 'pyodide';
+const toJsOptions = { dict_converter: Object.fromEntries };
```

Comment:
Perhaps it would be good to require a deep conversion with:
```js
toJsOptions = { ..., create_pyproxies: false }
```
and then in `transform` do:
```js
try {
   return value.toJs(toJsOptions);
} catch(e) {
   return value;
}
```

Reply:
Maybe this makes no sense though, I'm not sure.

Reply:
The current spotted use cases in the wild (Nicholas / Fabio) were all about passing a dict from Python to a `sync` callback defined in main. I don't like the suggested `try/catch` because it messes up with Pyodide specific possible errors and `postMessage` related error ... example:

  * a variable can't be converted due too many options
  * that variable is still posted as PyProxy and the logic fails

I think we should keep it simple for now, as it's tested as working as expected, and consider deeper/complex problems as we go.